source/memory.tex

```diff
@@ -864,6 +866,28 @@
 \end{note}
 \end{itemdescr}
 
+\indexlibraryglobal{assume_aligned}%
```

Comment:
I assume this was meant to align with the name of the new type, and is not sufficiently aligned?

Reply:
Fixed... ahem, aligned.